docker-compose.yml

```diff
@@ -0,0 +1,50 @@
+version: "3"
+
+services:
+  db:
+    image: postgres:12.1-alpine
+    ports:
+      - "5432:5432"
+    restart: always
+    environment:
+      POSTGRES_PASSWORD: password
+      POSTGRES_DB: transition
+
+  adminer:
+    image: adminer
+    restart: always
+    ports:
+      - 8888:8080
+
+  redis:
+    image: redis:4.0.14
+    restart: always
+    ports:
+      - 6379:6379
+
+  transition:
+    build: .
+    environment:
+      DATABASE_URL: postgres://postgres:password@db:5432/transition
+      REDIS_URL: redis://redis:6379
+      SECRET_KEY_BASE: 873834943a6232c50e6e515f99da526d6b3855b765d24de04688ecb26930fc5137b5abba7774785fa38db9bb681d326c37b162ac1c2860c8b95f1c4ffe2822d2
+    ports:
+      - "3000:3000"
+    depends_on:
+      - db
+      - redis
+    links:
+      - db
+      - redis
+
+  bouncer:
+    build: ../bouncer

```

Comment:
Gross

:)

Reply:
Yeah, if there's a better way (build image straight from github?) that would be good.

Reply:
the docker images all get built automatically so this could just use the bouncer image at https://hub.docker.com/repository/docker/thedxw/bouncer

Reply:
especially since bouncer changes less than transition.

Reply:
good idea